### ISSUE TITLE
CORS headers issue

### ISSUE THREAD

**Issue by Firsak** [codingforentrepreneurs/Tweetme-2#9]
Hello!

I was working through the project and everything worked just fine, but at the step of adding dev rest api authentication CORS header broke.
`Access to XMLHttpRequest at 'http://localhost:8000/api/tweets/' from origin 'http://127.0.0.1:3000' has been blocked by CORS policy: Request header field http_x_requested_with is not allowed by Access-Control-Allow-Headers in preflight response.`
It doesn't work both with the localhost and 127.0.0.1
I even dowloaded the full application of this step and it didn't work either.

**Comment by Firsak** [comment]
Hmmmmm, it's really strange, but I don't receive it in a new window...
<img width="1198" alt="Screenshot 2021-02-01 at 22 06 38" src="https://user-images.githubusercontent.com/31401457/106518182-ce7e5a00-64d9-11eb-86cd-91ed141dc5f6.png">

**Comment by rd9911** [comment]
Did you do all configurations in settings.py?

**Comment by Firsak** [comment]
> Did you do all configurations in settings.py?

Yes, I do. Because it was working till I added the dev authentication. 
And to be sure, I downloaded the version of the project from tweetme2 master branch.

As I understood it's some bug with the browsers, because I was suddenly receiving this error twice in the projects, that were already working.

**Comment by SnowLLL** [comment]
Did you try http://localhost:8000/?
